repo/retention_blob.go

```diff
@@ -0,0 +1,78 @@
+package repo
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// RetentionBlobID is the identifier of a BLOB that describes BLOB retention
+// settings for the repository.
+const RetentionBlobID = "kopia.retention"
+
+type retentionBlob struct {
+	Mode   string        `json:"mode,omitempty"`
```

Comment:
for future proofing, can this be `retentionMode` and `retentionPeriod` ?

Reply:
done, also renamed the struct

Reply:
f65715a